src/panels/LogPanel/ScriptLogPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useState } from 'react';
+import { ActionIcon, Group, Tooltip } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { FilterList } from '@/components/FilterList/FilterList';
+import { wordBeginningSubString } from '@/components/FilterList/util';
+import { RefreshIcon } from '@/icons/icons';
+
+import { ScriptLogEntry } from './ScriptLogEntry';
+
+// TODO (anden88 2025-04-17): Remove when lists from OpenSpace don't have the
+// ['1']: value, ['2']: value format
+type FileLines = Record<string, string> | undefined;
+
+export function ScriptLogPanel() {
+  const [scriptLogEntries, setScriptLogEntries] = useState<string[] | undefined>(
+    undefined
+  );
+  const luaApi = useOpenSpaceApi();
+
+  const fetchScriptLogEntries = useCallback(async () => {
+    // eslint-disable-next-line no-template-curly-in-string
+    const fileName = await luaApi?.absPath('${LOGS}/ScriptLog.txt');
+
+    if (!fileName) {
+      return;
+    }
+    const data = (await luaApi?.readFileLines(fileName)) as FileLines;
+
+    if (!data) {
+      return;
+    }
+
+    setScriptLogEntries(
+      Object.values(data)
+        .filter((script) => script)
+        .reverse()
+    );
+  }, [luaApi]);
+
+  useEffect(() => {
+    fetchScriptLogEntries();
+  }, [fetchScriptLogEntries]);
+
+  function preprocessScriptEntry(entry: string): string {
+    // Replace special characters with spaces. The '+' ensures repeated punctuation is
+    // replaced by a single space instead of many.
+    return entry.replace(/[.,()[\]{}"'\\/]+/g, ' ');
+  }
```

Comment:
"preprocess" does not seem like the right word here, IMO. Also, it does not say anything about what the function does, which is to "sanitize" the string

How about using "sanitize" or "clean" instead? Also, here, the `entry` is actually just a string for the script, so make that clearer in the name: 

```suggestion
  function sanitizeScript(script: string): string {
    // Replace special characters with spaces. The '+' ensures repeated punctuation is
    // replaced by a single space instead of many
    return script.replace(/[.,()[\]{}"'\\/]+/g, ' ');
  }
```

Reply:
With this new more descriptive name, it could also be moved to a utility file, if we want